stable/kubernetes-dashboard/templates/ingress.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+  {{- range $key, $value := .Values.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+  labels:
+    app: {{ template "name" . }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  name: {{ template "fullname" . }}
+spec:
+  rules:
+  {{- range .Values.ingress.hosts }}
+    - host: {{ . }}
+      http:
+        paths:
+          - path: /
+            backend:
+              serviceName: {{ template "fullname" . }}
+              servicePort: {{ .Values.httpPort }}
```

Comment:
When enabling Ingress with a hostname: The `.` inside this loop is one of the `hostname` strings defined in `values.yaml`. So both `template` and `.Values` fails with:

```
Error: render error in "kubernetes-dashboard/templates/ingress.yaml": template: kubernetes-dashboard/templates/_helpers.tpl:14:51: executing "fullname" at <.Values.nameOverride>: can't evaluate field Values in type string
```